vendor/fuser/src/lib.rs

```diff
@@ -26,7 +26,7 @@ use crate::mnt::mount_options::check_option_conflicts;
 use crate::session::MAX_WRITE_SIZE;
 #[cfg(feature = "abi-7-16")]
 pub use ll::fuse_abi::fuse_forget_one;
-pub use mnt::mount_options::MountOption;
+pub use mnt::{Mount, mount_options::MountOption};
```

Comment:
Yes I already updated our fork and used that script to bump up vendored fuser, see my commit [4eaf3e420a34f00f66b098f9e408904682ce4724](https://github.com/awslabs/mountpoint-s3/pull/1103/commits/4eaf3e420a34f00f66b098f9e408904682ce4724). Though it seems like it's already bumped on `main`, it should go away once I rebase.

Yes, this is just for tests. We still need to perform `mount` syscall in our tests and `Mount` provides this functionality with auto unmounting on drop and handling different libfuse versions. So this small divergence from upstream seemed better to me instead of duplicating the whole logic.